items:
- uid: system.fabric.exception._fabric_delete_backup_file_failed_exception
  id: _fabric_delete_backup_file_failed_exception
  parent: system.fabric.exception
  children:
  - system.fabric.exception._fabric_delete_backup_file_failed_exception.FabricDeleteBackupFileFailedException()
  - system.fabric.exception._fabric_delete_backup_file_failed_exception.FabricDeleteBackupFileFailedException(String)
  - system.fabric.exception._fabric_delete_backup_file_failed_exception.FabricDeleteBackupFileFailedException(String,Throwable)
  - system.fabric.exception._fabric_delete_backup_file_failed_exception.FabricDeleteBackupFileFailedException(String,Throwable,boolean,boolean)
  - system.fabric.exception._fabric_delete_backup_file_failed_exception.FabricDeleteBackupFileFailedException(Throwable)
  href: system.fabric.exception._fabric_delete_backup_file_failed_exception.yml
  langs:
  - java
  name: FabricDeleteBackupFileFailedException
  fullName: system.fabric.exception.FabricDeleteBackupFileFailedException
  type: Class
  source:
    remote: &o0
      path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricDeleteBackupFileFailedException.java
      branch: release_linux_5.3
      repo: https://msazure.visualstudio.com/One/_git/sf-java
    path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricDeleteBackupFileFailedException.java
    startLine: 4
  package: system.fabric.exception
  syntax: &o1
    content: public class FabricDeleteBackupFileFailedException extends FabricException
  inheritance:
  - java.lang.Object
  - RuntimeException
  - system.fabric.exception._fabric_exception
  inheritedMembers:
  - system.fabric.exception._fabric_exception.FabricException()
  - system.fabric.exception._fabric_exception.FabricException(FabricErrorCode)
  - system.fabric.exception._fabric_exception.FabricException(String)
  - system.fabric.exception._fabric_exception.FabricException(String,FabricErrorCode)
  - system.fabric.exception._fabric_exception.FabricException(String,Throwable)
  - system.fabric.exception._fabric_exception.FabricException(String,Throwable,FabricErrorCode)
  - system.fabric.exception._fabric_exception.FabricException(Throwable)
  - system.fabric.exception._fabric_exception.FabricException(Throwable,FabricErrorCode)
  - system.fabric.exception._fabric_exception.FabricException(String,Throwable,boolean,boolean)
  - system.fabric.exception._fabric_exception.FabricException(String,Throwable,boolean,boolean,FabricErrorCode)
  - system.fabric.exception._fabric_exception.getErrorCode()
- uid: system.fabric.exception._fabric_delete_backup_file_failed_exception.FabricDeleteBackupFileFailedException()
  id: FabricDeleteBackupFileFailedException()
  parent: system.fabric.exception._fabric_delete_backup_file_failed_exception
  href: system.fabric.exception._fabric_delete_backup_file_failed_exception.yml
  langs:
  - java
  name: FabricDeleteBackupFileFailedException()
  fullName: system.fabric.exception.FabricDeleteBackupFileFailedException.FabricDeleteBackupFileFailedException()
  type: Constructor
  source:
    remote: *o0
    path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricDeleteBackupFileFailedException.java
    startLine: 10
  package: system.fabric.exception
  syntax:
    content: public FabricDeleteBackupFileFailedException()
- uid: system.fabric.exception._fabric_delete_backup_file_failed_exception.FabricDeleteBackupFileFailedException(String)
  id: FabricDeleteBackupFileFailedException(String)
  parent: system.fabric.exception._fabric_delete_backup_file_failed_exception
  href: system.fabric.exception._fabric_delete_backup_file_failed_exception.yml
  langs:
  - java
  name: FabricDeleteBackupFileFailedException(String message)
  fullName: system.fabric.exception.FabricDeleteBackupFileFailedException.FabricDeleteBackupFileFailedException(String message)
  type: Constructor
  source:
    remote: *o0
    path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricDeleteBackupFileFailedException.java
    startLine: 35
  package: system.fabric.exception
  summary: >-
    <p>Initializes a new instance of the <xref href="system.fabric.exception._fabric_cannot_connect_exception" data-throw-if-not-resolved="false"></xref> class with a specified error message.</p>

    <p></p>
  syntax:
    content: public FabricDeleteBackupFileFailedException(String message)
    parameters:
    - id: message
      type: "26831127"
      description: <p>The error message that explains the reason for the exception. </p>
- uid: system.fabric.exception._fabric_delete_backup_file_failed_exception.FabricDeleteBackupFileFailedException(String,Throwable)
  id: FabricDeleteBackupFileFailedException(String,Throwable)
  parent: system.fabric.exception._fabric_delete_backup_file_failed_exception
  href: system.fabric.exception._fabric_delete_backup_file_failed_exception.yml
  langs:
  - java
  name: FabricDeleteBackupFileFailedException(String message, Throwable cause)
  fullName: system.fabric.exception.FabricDeleteBackupFileFailedException.FabricDeleteBackupFileFailedException(String message, Throwable cause)
  type: Constructor
  source:
    remote: *o0
    path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricDeleteBackupFileFailedException.java
    startLine: 24
  package: system.fabric.exception
  summary: >-
    <p>Initializes a new instance of the <xref href="system.fabric.exception._fabric_delete_backup_file_failed_exception" data-throw-if-not-resolved="false"></xref> class with a specified error message and error code.</p>

    <p></p>
  syntax:
    content: public FabricDeleteBackupFileFailedException(String message, Throwable cause)
    parameters:
    - id: message
      type: "26831127"
      description: <p>The error message that explains the reason for the exception. </p>
    - id: cause
      type: a04fafd7
      description: <p>The exception that is the cause of the current exception, or a null reference if no inner exception is specified. </p>
- uid: system.fabric.exception._fabric_delete_backup_file_failed_exception.FabricDeleteBackupFileFailedException(String,Throwable,boolean,boolean)
  id: FabricDeleteBackupFileFailedException(String,Throwable,boolean,boolean)
  parent: system.fabric.exception._fabric_delete_backup_file_failed_exception
  href: system.fabric.exception._fabric_delete_backup_file_failed_exception.yml
  langs:
  - java
  name: FabricDeleteBackupFileFailedException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
  fullName: system.fabric.exception.FabricDeleteBackupFileFailedException.FabricDeleteBackupFileFailedException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
  type: Constructor
  source:
    remote: *o0
    path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricDeleteBackupFileFailedException.java
    startLine: 43
  package: system.fabric.exception
  syntax:
    content: protected FabricDeleteBackupFileFailedException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
    parameters:
    - id: message
      type: "26831127"
    - id: cause
      type: a04fafd7
    - id: enableSuppression
      type: 4fc6e284
    - id: writableStackTrace
      type: 4fc6e284
- uid: system.fabric.exception._fabric_delete_backup_file_failed_exception.FabricDeleteBackupFileFailedException(Throwable)
  id: FabricDeleteBackupFileFailedException(Throwable)
  parent: system.fabric.exception._fabric_delete_backup_file_failed_exception
  href: system.fabric.exception._fabric_delete_backup_file_failed_exception.yml
  langs:
  - java
  name: FabricDeleteBackupFileFailedException(Throwable cause)
  fullName: system.fabric.exception.FabricDeleteBackupFileFailedException.FabricDeleteBackupFileFailedException(Throwable cause)
  type: Constructor
  source:
    remote: *o0
    path: src/prod/src/java/system.fabric/src/system/fabric/exception/FabricDeleteBackupFileFailedException.java
    startLine: 39
  package: system.fabric.exception
  syntax:
    content: public FabricDeleteBackupFileFailedException(Throwable cause)
    parameters:
    - id: cause
      type: a04fafd7
references:
- uid: "26831127"
  spec.java:
  - name: String
    fullName: String
- uid: a04fafd7
  spec.java:
  - name: Throwable
    fullName: Throwable
- uid: 4fc6e284
  spec.java:
  - name: boolean
    fullName: boolean
- uid: system.fabric.exception._fabric_exception.FabricException()
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException()
  fullName: system.fabric.exception.FabricException.FabricException()
  type: Constructor
  syntax:
    content: public FabricException()
- uid: system.fabric.exception._fabric_exception.FabricException(FabricErrorCode)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(FabricErrorCode errorCode)
  fullName: system.fabric.exception.FabricException.FabricException(FabricErrorCode errorCode)
  type: Constructor
  syntax:
    content: public FabricException(FabricErrorCode errorCode)
    parameters:
    - id: errorCode
      type: system.fabric._fabric_error_code
- uid: system.fabric.exception._fabric_exception.FabricException(String)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(String message)
  fullName: system.fabric.exception.FabricException.FabricException(String message)
  type: Constructor
  syntax:
    content: public FabricException(String message)
    parameters:
    - id: message
      type: "26831127"
- uid: system.fabric.exception._fabric_exception.FabricException(String,FabricErrorCode)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(String message, FabricErrorCode errorCode)
  fullName: system.fabric.exception.FabricException.FabricException(String message, FabricErrorCode errorCode)
  type: Constructor
  syntax:
    content: public FabricException(String message, FabricErrorCode errorCode)
    parameters:
    - id: message
      type: "26831127"
    - id: errorCode
      type: system.fabric._fabric_error_code
- uid: system.fabric.exception._fabric_exception.FabricException(String,Throwable)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(String message, Throwable cause)
  fullName: system.fabric.exception.FabricException.FabricException(String message, Throwable cause)
  type: Constructor
  syntax:
    content: public FabricException(String message, Throwable cause)
    parameters:
    - id: message
      type: "26831127"
    - id: cause
      type: a04fafd7
- uid: system.fabric.exception._fabric_exception.FabricException(String,Throwable,FabricErrorCode)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(String message, Throwable cause, FabricErrorCode errorCode)
  fullName: system.fabric.exception.FabricException.FabricException(String message, Throwable cause, FabricErrorCode errorCode)
  type: Constructor
  syntax:
    content: public FabricException(String message, Throwable cause, FabricErrorCode errorCode)
    parameters:
    - id: message
      type: "26831127"
    - id: cause
      type: a04fafd7
    - id: errorCode
      type: system.fabric._fabric_error_code
- uid: system.fabric.exception._fabric_exception.FabricException(Throwable)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(Throwable cause)
  fullName: system.fabric.exception.FabricException.FabricException(Throwable cause)
  type: Constructor
  syntax:
    content: public FabricException(Throwable cause)
    parameters:
    - id: cause
      type: a04fafd7
- uid: system.fabric.exception._fabric_exception.FabricException(Throwable,FabricErrorCode)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(Throwable cause, FabricErrorCode errorCode)
  fullName: system.fabric.exception.FabricException.FabricException(Throwable cause, FabricErrorCode errorCode)
  type: Constructor
  syntax:
    content: public FabricException(Throwable cause, FabricErrorCode errorCode)
    parameters:
    - id: cause
      type: a04fafd7
    - id: errorCode
      type: system.fabric._fabric_error_code
- uid: system.fabric.exception._fabric_exception.FabricException(String,Throwable,boolean,boolean)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
  fullName: system.fabric.exception.FabricException.FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
  type: Constructor
  syntax:
    content: protected FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace)
    parameters:
    - id: message
      type: "26831127"
    - id: cause
      type: a04fafd7
    - id: enableSuppression
      type: 4fc6e284
    - id: writableStackTrace
      type: 4fc6e284
- uid: system.fabric.exception._fabric_exception.FabricException(String,Throwable,boolean,boolean,FabricErrorCode)
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace, FabricErrorCode errorCode)
  fullName: system.fabric.exception.FabricException.FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace, FabricErrorCode errorCode)
  type: Constructor
  syntax:
    content: protected FabricException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace, FabricErrorCode errorCode)
    parameters:
    - id: message
      type: "26831127"
    - id: cause
      type: a04fafd7
    - id: enableSuppression
      type: 4fc6e284
    - id: writableStackTrace
      type: 4fc6e284
    - id: errorCode
      type: system.fabric._fabric_error_code
- uid: system.fabric.exception._fabric_exception.getErrorCode()
  parent: system.fabric.exception._fabric_exception
  href: system.fabric.exception._fabric_exception.yml
  name: getErrorCode()
  fullName: FabricErrorCode system.fabric.exception.FabricException.getErrorCode()
  type: Method
  syntax:
    content: public FabricErrorCode getErrorCode()
    return:
      type: system.fabric._fabric_error_code
- uid: system.fabric.exception._fabric_exception
  parent: system.fabric.exception
  href: system.fabric.exception._fabric_exception.yml
  name: FabricException
  fullName: system.fabric.exception.FabricException
  type: Class
  syntax:
    content: public class FabricException
- uid: system.fabric.exception._fabric_delete_backup_file_failed_exception
  parent: system.fabric.exception
  href: system.fabric.exception._fabric_delete_backup_file_failed_exception.yml
  name: FabricDeleteBackupFileFailedException
  fullName: system.fabric.exception.FabricDeleteBackupFileFailedException
  type: Class
  syntax: *o1
- uid: system.fabric.exception._fabric_cannot_connect_exception
  parent: system.fabric.exception
  href: system.fabric.exception._fabric_cannot_connect_exception.yml
  name: FabricCannotConnectException
  fullName: system.fabric.exception.FabricCannotConnectException
  type: Class
  syntax:
    content: public class FabricCannotConnectException extends FabricException
